AK/Format.cpp

```diff
@@ -394,7 +394,7 @@ void FormatBuilder::put_f80(
     StringBuilder string_builder;
```

Comment:
> Everywhere: Use the correct suffixes and format strings

This is a bit of a mixed commit, but at least call it "printf format strings", as the current one is too generic.